src/main/java/homework/collections/hospital/repository/HospitalRepositoryImpl.java

```diff
@@ -0,0 +1,133 @@
+package homework.collections.hospital.repository;
+
+import homework.collections.hospital.exceptions.DoctorHasntAvailableTimeException;
+import homework.collections.hospital.exceptions.PatientDontPaidException;
+import homework.collections.hospital.exceptions.ReservationNotFoundException;
+import homework.collections.hospital.model.Doctor;
+import homework.collections.hospital.model.Patient;
+import homework.collections.hospital.model.Time;
+
+import java.util.*;
+
+public class HospitalRepositoryImpl implements HospitalRepository<Doctor, Patient, Time> {
+
+    private static final Map<Doctor, Map<Time, List<Patient>>> repository = new HashMap();
```

Comment:
then you need create HospitalRepository class, not interface. And add this map there.
public static final Map<Doctor, Map<Time, List<Patient>>> repository = new HashMap();